src/app/api/run/[runId]/upload/route.test.ts

```diff
@@ -55,4 +95,27 @@ describe('POST /api/run/[runId]/upload', () => {
         const response = await POST(req, { params })
         expect(response.status).toBe(400)
     })
+
+    it('should return an error if the runID has results already', async () => {
```

Comment:
one "product" question i have around this is
1. will they ever be uploading results for the same run? ie runs will always have different uuids right? so this is just a safeguard
2. if the above is true, ie they can upload results for the same run, would we want to override and upload the newer results? ie overwrite vs error



just thinking out loud 🤔 💭

Reply:
Ah, I see. Yeah, my implicit assumption currently is it's a safeguard in case the researcher code (for some reason that was missed during code review :sweat_smile: ) attempts to submit multiple times for the same run.

Reply:
yeah with your 2nd comment in mind, happy to just reject it and enforce a new runid upload :)